CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [3.0.0] - 2020-02-25
+### Removed
+
+* Dropped support for ^1.0 releases of HTTPlug (phphttp/httplug)
+    * HTTP clients must now use the `Psr\Http\Client\ClientInterface` (found in 
+    ^2.0 releases of HTTPlug) rather than `Http\Client\HttpClient` interface 
+    (found in ^1.0 releases of HTTPlug).
```

Comment:
```suggestion
    * HTTP clients may now use the `Psr\Http\Client\ClientInterface` (found in 
    ^2.0 releases of HTTPlug) rather than `Http\Client\HttpClient` interface 
    (found in ^1.0 releases of HTTPlug).
```

Reply:
I think a may is appropriate as in ^2.0 they essentially alias HttpClient to ClientInterface